### ISSUE TITLE
Low mIoU, low loss

### ISSUE THREAD

**Issue by adhitir** [DeepSceneSeg/AdapNet#3]
I followed the steps in your readme. I'm using the Freiburg forest dataset -- I augmented the images from the rgb, depth and evi data and created the text file for training and ran train.py with the resnet model as the initial checkpoint. The loss function starts at 0.0034-ish and drops to 0.0000. 
Evaluate.py gives me an mIoU of 12.64. 
I'm not sure what I'm doing incorrectly.

**Comment by avalada** [comment]
Please make sure that the groundtruth is mapped correctly.
